source/index.md

```diff
@@ -807,13 +807,13 @@ https://gateway.clearhaus.com/account
 
 ## Changes
 
-### CLP and UGX exponent change
+### CLP and UGX exponent changes
 
-CLP and UGX changes from exponent 2 to exponent 2.
+CLP and UGX changes from exponent 2 to exponent 0.
```

Comment:
I was actually referring to "changes" after "CLP and UGX" :wink: However, I like the change in the section title :slightly_smiling_face:

Reply:
I'm lost. Which line? What diff?